completions/fish/crio.fish

```diff
@@ -178,6 +178,9 @@ by CRI-O. This allows you to save you current configuration setup and then load
 it later with **--config**. Global options will modify the output.'
 complete -c crio -n '__fish_seen_subcommand_from config' -f -l default -d 'Output the default configuration (without taking into account any configuration options).'
 complete -c crio -n '__fish_seen_subcommand_from config' -f -l migrate-defaults -s m -r -d 'Migrate the default config from a specified version.
+
+    The migrate-defaults command has been deprecated and will be removed in v1.29.0.
```

Comment:
sorry to add another nit: we don't usually set the exact version we'll be deprecating in case we're not correct. in this case, it isn't too risky because i don't see much that will cause us to not drop in 1.29.0, but I am leaning towards keeping with convention here. we usually say "deprecated and will be removed in the future"

Reply:
updated